torch/_torch_docs.py

```diff
@@ -5298,6 +5298,40 @@ def merge_dicts(*dicts):
 
 """.format(**common_args))
 
+add_docstr(torch.heaviside,
+           r"""
+heaviside(input, val, *, out=None) -> Tensor
+
+Computes the Heaviside step function for each element in :attr:`input`.
+
+The Heaviside step function is defined as:
+
+.. math::
+    \text{{heaviside}}(input, val) = \begin{cases}
+        \0, & \text{if input < 0}\\
+        \val, & \text{if input == 0}\\
+        \1, & \text{if input > 0}
+    \end{cases}
+""" + r"""
+Args:
+    {input}
+    val (Float point Tensor or value): The value of the element where is 0 in input. 
```

Comment:
"The value or values to sample from where :attr:\`input\` is zero." ?

Reply:
Addressed!